lib/logstash/outputs/elasticsearch.rb

```diff
@@ -456,9 +456,7 @@ def get_event_type(event)
              event.sprintf(@document_type)
            else
              major_version = maximum_seen_major_version
-             if major_version < 6
-               es5_event_type(event)
```

Comment:
believe the (private) `es5_event_type(event)` method could be removed as well

Reply:
🤦 removal of that method was unstaged in my local repo, solved